src/components/TagsPoplist/index.tsx

```diff
@@ -60,6 +60,15 @@ type TagsPoplistProps = {
   multiline?: boolean
 }
 
+const TagContainer = styled.div<{ index: number; visibleTagsCount: number }>`
+  margin-right: ${({ index, visibleTagsCount }) =>
+    index + 1 !== visibleTagsCount ? 1 : 0};
+`
+
+const StyledTag = styled(Tag)`
+  margin-right: ${({ theme }) => theme.space['1']};
+`
+
```

Comment:
Instead of always playing with margin I think we could use a gap somewhere :)

Reply:
done